### PR TITLE
Moving towards NetStandard

### PR DESCRIPTION
**Pull request by alexvaluyskiy**
Fixed https://github.com/akkadotnet/akka.net/issues/2153

### CODE REVIEW

**Comment by Aaronontheweb** [comment]
To be clear, the build process for this system WILL NOT RUN right now. Need to add dotnetcore support to the build agents to do that.

**Comment by adamhathcock** [comment]
I haven't read everything as it's late in my day but just wanted to ask: do you think you should make the libraries target as low as possible (like standard 1.3) instead of 1.6?

It practically may not matter because you have net45 there and 1.3 is net451 but I think best practice is going as low as possible.

The PCL compatibility package is new to me so I should use that.  Thanks :)

File: src/core/Akka.Tests/Actor/Dispatch/ActorModelSpec.cs [hunk 1]
```diff
@@ -218,7 +218,16 @@ public DispatcherActor()
...
                 Receive<Interrupt>(interrupt => { Ack(); Sender.Tell(new Status.Failure(new ActorInterruptedException(cause: new Exception(Ping)))); _busy.SwitchOff(); throw new Exception(Ping); }, interrupt => true);
-                Receive<InterruptNicely>(interrupt => { Ack(); Sender.Tell(interrupt.Expect); _busy.SwitchOff(); Thread.CurrentThread.Interrupt(); });
+                Receive<InterruptNicely>(interrupt =>
+                {
+                    Ack();
+                    Sender.Tell(interrupt.Expect);
+                    _busy.SwitchOff();
+#if !CORECLR
+                    // TODO: CORECLR FIX IT
+                    Thread.CurrentThread.Interrupt();
```
Comment (alexvaluyskiy) [inline src/core/Akka.Tests/Actor/Dispatch/ActorModelSpec.cs]: Netcore doesn't have Thread.Interupt

File: src/core/Akka/Actor/Props.cs [hunk 2]
```diff
@@ -763,7 +764,12 @@ private DynamicProps(Props copy, Func<TActor> invoker)
         protected override Props Copy()
         {
             Props initialCopy = base.Copy();
+#if CLONEABLE
             var invokerCopy = (Func<TActor>)invoker.Clone();
+#else
+            // TODO: CORECLR FIX IT
+            var invokerCopy = (Func<TActor>)invoker;
```
Comment (alexvaluyskiy) [inline src/core/Akka/Actor/Props.cs]: Should fix it

**Comment by annymsMthd** [comment]
We should probably have these commits squashed

**Comment by Aaronontheweb** [comment]
@alexvaluyskiy go ahead and squash these and we can merge them in.

**Pull request merged**